filebeat/module/kafka/log/ingest/pipeline.json

```diff
@@ -6,18 +6,18 @@
         "field": "message",
         "trace_match": true,
         "patterns": [
-          "(?m)%{TIMESTAMP_ISO8601:kafka.log.timestamp}. %{LOGLEVEL:kafka.log.level} +%{JAVALOGMESSAGE:kafka.log.message} \\(%{JAVACLASS:kafka.log.class}\\)$[ \\n]*(?'kafka.log.trace.full'.*)"
+          "(?m)%{TIMESTAMP_ISO8601:kafka.log.timestamp}. %{LOGLEVEL:log.level} +%{JAVALOGMESSAGE:message} \\(%{JAVACLASS:kafka.log.class}\\)$[ \\n]*(?'kafka.log.trace.full'.*)"
```

Comment:
Sounds like one day we should have something related to Java class.

@felixbarny Is there a standardised way Java APM agent is storing this info?